python/tk_multi_workfiles/scene_operation.py

```diff
@@ -107,7 +106,7 @@ def get_current_path(app, action, context):
     """
     app.log_debug("Retrieving current scene path...")
     return _do_scene_operation(
-        app, action, context, "current_path", result_types=(six.string_types,)
+        app, action, context, "current_path", result_types=((str,),)
```

Comment:
I am not sure about the type we return here?!

Where is this thing used? Is it really expending a list of list of types?

Reply:
fixed, [_do_scene_operation](https://github.com/shotgunsoftware/tk-multi-workfiles2/blob/master/python/tk_multi_workfiles/scene_operation.py#L47) requires only a tuple